# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qemu/no-tests.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

superh-24.12-*/tests/fp/fp-bench.c: In function 'set_host_precision':
superh-24.12.*/tests/fp/fp-bench.c:568:17: error: 'FE_DOWNWARD' undeclared (first use in this function); did you mean 'FP_INT_DOWNWARD'?

--- qemu-9.1.1/meson.build.vanilla	2024-12-05 16:55:31.743195941 +0100
+++ qemu-9.1.1/meson.build	2024-12-05 16:57:36.619861819 +0100
@@ -4168,7 +4168,7 @@
 subdir('tools')
 subdir('pc-bios')
 subdir('docs')
-subdir('tests')
+#subdir('tests')
 if gtk.found()
   subdir('po')
 endif
